src/client/datascience/kernel-launcher/localKernelFinder.ts

```diff
@@ -176,13 +180,19 @@ export class LocalKernelFinder implements ILocalKernelFinder {
         resource: Resource,
         cancelToken?: CancellationToken
     ): Promise<LocalKernelConnectionMetadata[]> {
+        traceInfo('IANHU');
         // First find the on disk kernel specs and interpreters
-        const [kernelSpecs, interpreters, rootSpecPath, activeInterpreter] = await Promise.all([
-            this.findResourceKernelSpecs(resource, cancelToken),
-            this.findResourceInterpreters(resource, cancelToken),
-            this.getKernelSpecRootPath(),
-            this.getActiveInterpreter(resource)
-        ]);
+        const activeInterpreter = await this.getActiveInterpreter(resource);
```

Comment:
Uh not sure why you'd do these in order instead of at the same time?

Reply:
Was something not working?

Reply:
@rchiodo Sorry you are not on the teams chat. It's based off a junk branch that I was doing some kernel work in, just to get the correct proposed API changes (which I had already done in that branch). I'm going to port this after my breaking changes fix is in.